src/Adapter/Product/Combination/CommandHandler/UpdateCombinationStockHandler.php

```diff
@@ -43,22 +45,38 @@ final class UpdateCombinationStockHandler implements UpdateCombinationStockHandl
      */
     private $combinationStockUpdater;
 
+    /**
+     * @var MovementReasonRepository
+     */
+    private $movementReasonRepository;
+
     /**
      * @param CombinationStockUpdater $combinationStockUpdater
+     * @param MovementReasonRepository $movementReasonRepository
      */
     public function __construct(
-        CombinationStockUpdater $combinationStockUpdater
+        CombinationStockUpdater $combinationStockUpdater,
+        MovementReasonRepository $movementReasonRepository
     ) {
         $this->combinationStockUpdater = $combinationStockUpdater;
+        $this->movementReasonRepository = $movementReasonRepository;
     }
 
     /**
      * {@inheritdoc}
      */
     public function handle(UpdateCombinationStockCommand $command): void
     {
+        $deltaQty = null;
+        if (null !== $command->getDeltaQuantity()) {
```

Comment:
```suggestion
        if (null === $command->getDeltaQuantity()) {
                return;
        }
```

Reply:
this is a little different than above example.
I need to proceed as the following code does not depend only from quantity, but there are other updatable props. (e.g. one may update only the stock location, but not quantity)

Reply:
Yes, you're right! Thanks